examples/pset_blind_coinjoin.rs

```diff
@@ -287,3 +285,43 @@ fn main() {
     tx.verify_tx_amt_proofs(&secp, &[btc_txout, asset_txout])
         .unwrap();
 }
+
+
+/// Xorshift
+pub struct CrappyRng(u64);
```

Comment:
I took a simple PRNG from Wikipedia,

for laziness it fail something like

rng = CrappyRng(1);
rng = CrappyRng(1);
`assert_eq!( 
concat!( rng.fill_bytes( 2_bytes_buffer), rng.fill_bytes( 2_bytes_buffer)), 
rng2.fill_bytes(4_bytes_buffer))`

I don't know if it matters here or in general

Reply:
Can you link the wikipedia page in the comments?

My impulse would've been to use a LCG, eg any of the params from https://en.wikipedia.org/wiki/Linear_congruential_generator#Parameters_in_common_use but if this one is on wikipedia somewhere I'm sure it's just as good :P.

I don't think that invariant matters at all .. is it supposed to hold for `RngCore`? Why would it be your responsibility to make it hold when you only implement one method?

Reply:
> Can you link the wikipedia page in the comments?

To start I post it here https://en.wikipedia.org/wiki/Xorshift#Example_implementation

> I  don't think that invariant matters at all .. is it supposed to hold for RngCore? 

No, I found in the doc:
> many implementations of [fill_bytes](https://docs.rs/rand/latest/rand/trait.RngCore.html#tymethod.fill_bytes) consume a whole number of u32 or u64 values and drop any remaining unused bytes